src/vision/QAK/transform_data.py

```diff
@@ -0,0 +1,15 @@
+import reading_data as rd
+import numpy as np
+import open3d as o3d
+import cv2
+import os
+
+
+def make_sectors(x, y):
+    # make grid for sectors to be used in 3d data transformation to 2d
+    pass
+
+
+def assignment_to_sectors(sectors, pcd):
+    # assign 3d points to 2d sectors
+    pass
```

Comment:
jak mergowałś z mojego brancha to to nie push'uj tych zmian z swojego bo będziemy mieli konflikty zrób sobie `git rm -r --cached <file>`